recipes/neptune-client/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "neptune-client" %}
+{% set version = "0.4.99" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/neptune-ml/neptune-client/archive/{{ version }}.tar.gz"
+  sha256: "6e4da58830d65c483e915fd3944da358e869c0085293c876e0be4d33ab6f0a5d"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - bravado
+    - bravado-core
+    - click >=7.0
+    - future >=0.17.1
+    - gitpython >=2.0.8
+    - py3nvml
+    - oauthlib >=2.1.0
+    - pandas
+    - pillow >=1.1.6
+    - pyjwt
+    - python
+    - requests >=2.20.0
+    - requests-oauthlib >=1.0.0
+    - six >=1.12.0
+    - websocket-client >=0.35.0
+
+test:
+  imports:
+    - neptune
+    - neptune.internal
+    - neptune.internal.backends
+    - neptune.internal.channels
+    - neptune.internal.execution
+    - neptune.internal.hardware
+    - neptune.internal.hardware.cgroup
+    - neptune.internal.hardware.gauges
+    - neptune.internal.hardware.gpu
+    - neptune.internal.hardware.metrics
+    - neptune.internal.hardware.metrics.reports
+    - neptune.internal.hardware.metrics.service
+    - neptune.internal.hardware.resources
+    - neptune.internal.hardware.system
+    - neptune.internal.notebooks
+    - neptune.internal.storage
+    - neptune.internal.streams
+    - neptune.internal.threads
+    - neptune.internal.utils
+    - neptune.internal.websockets
+    - tests.neptune
+    - tests.neptune.internal
+    - tests.neptune.internal.backends
+    - tests.neptune.internal.channels
+    - tests.neptune.internal.hardware
+    - tests.neptune.internal.hardware.gauges
+    - tests.neptune.internal.hardware.metrics
+    - tests.neptune.internal.hardware.resources
+    - tests.neptune.internal.storage
+    - tests.neptune.utils
+
+about:
+  home: "https://neptune.ai/"
+  license: "Apache Software"
+  license_family: "APACHE"
+  license_file: "LICENSE"
+  summary: "Neptune Client"
+  doc_url: "https://github.com/neptune-ml/neptune-client/blob/master/README.md"
+  dev_url: "https://github.com/neptune-ml/neptune-client"
+
+extra:
+  recipe-maintainers:
+    - szymon-kuklewicz
+    - aniezurawski
+    - hubertjaworskineptune
+    - pitercl
```

Comment:
@pitercl @aniezurawski are you OK with being maintainers?

Reply:
Maintainers confirmed